### ISSUE TITLE
[Feature Request] Menu isn't accessible

### ISSUE THREAD

**Issue by t1gu1** [ElemeFE/element#16697]
### Existing Component
Yes

### Component Name
NavMenu
https://element.eleme.io/#/en-US/component/menu

### Description
The navigation menu isn't accessible.

The navingation with menu should be like that: 
https://www.w3.org/TR/wai-aria-practices/examples/menubar/menubar-1/menubar-1.html

As you can see, it's possible to navigate in the menu with the arrow.

- Left/Right, navigate in nav
- Up/Down, Navigate vertically in the focused menu

At least, it must be something navigable easly with tabulation without difficulty.

An accessible menu is a basic and important part in accessibility.

**Comment by ziyoung** [comment]
pr welcome.
